### PR TITLE
add AllReviews and ReviewBox

### PR DESCRIPTION
**Pull request by Freeman-YFH**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,10 +1,14 @@
...
+import { AllReviews } from './components/AllReviews';
+import { Header } from './components/Header';
+import { ReviewBox } from './components/ReviewBox';
 
 function App() {
   return (
     <div className="App">
-
+      <Header />
+      <AllReviews />
```
Comment (popatre) [inline src/App.js]: Could you set these up with routes/urls please? Presume this would be `/`

File: src/components/AllReviews.jsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+            setIsLoading(false);
+        })
+    }, []);
+
+    return (
+        <main>
+            <h2>Reviews List:</h2>
+            <ol className="all-reviews">
+                {reviews.map(review => {
+                    return <ReviewBox key={review.review_id} {...review} />
```
Comment (popatre) [inline src/components/AllReviews.jsx:28]: nice components

File: src/components/AllReviews.jsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
+import { useEffect, useState } from "react";
+import * as api from "./api";
+import { ReviewBox } from "./ReviewBox";
+
+export const AllReviews = () => {
+    const [reviews, setReviews] = useState([]);
+    const [isLoading, setIsLoading] = useState(true);
```
Comment (popatre) [inline src/components/AllReviews.jsx:7]: You've got your loading state that would done use.
COuld we get some feedback for when its loading on the page?

**Review by popatre — CHANGES_REQUESTED** [review]
Could you start setting up your Routes please?
And use your loading state to display some loading on the page? Just so we know something happening
Rest looks good

File: src/components/Login.jsx [hunk 4]
```diff
@@ -0,0 +1,11 @@
+import { Link } from "react-router-dom";
+
+export const Login = () => {
+    return (
+        <div>
+            <h2>Login page</h2>
+            <h2>Under construction</h2>
+            <Link to={'/allReviews'}>Reviews</Link>
```
Comment (itskatherine) [inline src/components/Login.jsx:8]: Nice addition of routing

**Review by itskatherine — APPROVED** [review]
Nice addition of routes and loading states Freeman, good to merge 👍

**Pull request merged**